ResourceManager/Xliff/TranslationUtil.cs

```diff
@@ -190,6 +207,26 @@ public static void TranslateItemsFromList(string category, ITranslation translat
                     continue;
                 }
 
+                if (itemObj is ToolTip tooltip)
+                {
+                    string ProvideDefaultValue() => "";
```

Comment:
```suggestion
                    static string ProvideDefaultValue() => "";
```

Reply:
Why not `=> null`?

Reply:
I have hesitated to change it when copy/pasting from previous definition. So no real reason except the copy/paste. So changed...